scripts/pylint_extensions.py

```diff
@@ -1197,6 +1197,7 @@ def process_module(self, node):
         """
 
         in_multi_line_comment = False
+        space_at_beginning = True
```

Comment:
Clarify in the variable name whether this is space_at_beginning_of_file or space_at_beginning_of_line or space_at_beginning_of_comment (or something else).'

Also is this a per-line variable? If so why are you declaring it here (and not within the for loop)?

Reply:
Done.

Reply:
Done.